### DISCUSSION TITLE
Legend

### DISCUSSION THREAD

**Discussion by Rozerinay** [nl-design-system/backlog#279] [Component suggestions]
<img width="480" alt="visuele weergave van de legend component" src="https://github.com/user-attachments/assets/6bbd9f44-d7e7-42f9-a5f6-8cad72cea237">

## Naam
Legend

## Doel
Toont een beschrijvende titel voor een groep gerelateerde formulier elementen.

## Gerelateerde componenten
[Fieldset](https://github.com/orgs/nl-design-system/discussions/309), [Form field label](https://github.com/orgs/nl-design-system/discussions/226)

## Onderzoek

**Comment by jeffreylauwers** [comment]
Zoals besproken in de [Backlog issue](https://github.com/nl-design-system/backlog/issues/186) gaan we hier geen los component van maken.
